selfdrive/car/hyundai/interface.py

```diff
@@ -228,9 +249,9 @@ def update(self, c, can_strings):
     ret.seatbeltUnlatched = not self.CS.seatbelt
 
     # low speed steer alert hysteresis logic (only for cars with steer cut off above 10 m/s)
-    if ret.vEgo < (self.CP.minSteerSpeed + 2.) and self.CP.minSteerSpeed > 10.:
+    if ret.vEgo < (self.CP.minSteerSpeed + 1.) and self.CP.minSteerSpeed > 10.:
       self.low_speed_alert = True
-    if ret.vEgo > (self.CP.minSteerSpeed + 4.):
+    if ret.vEgo > (self.CP.minSteerSpeed + 1.):
```

Comment:
don;t you want an hysteresis?

Reply:
it is very annoying to get "TAKE CONTROL, steering unavailable" while driving 9 mph above min steer speed
I would like to remove this "+ 1." in order for Eon instructions to be accurate

Reply:
I agree that +2 and +4 was too much. But some hysteresis is needed I think. perhaps 0.5 and 1.?

Reply:
I just did a search and it turned out that Hyundai is the only car uses this hysteresis logic which mean it is not necessary 
https://github.com/commaai/openpilot/blob/069e337bead4b8354af7e8b73f2705d792e90a32/selfdrive/car/chrysler/interface.py#L183
as a user, I can't see any benefit from it. 
if you agree I will be happy to remove it otherwise, I could change it to 0.5 to start and 0.7 to exit or whatever you recommend